client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -279,53 +293,114 @@ class CampaignAdminList extends Component {
     }
     return (
       <div className="btn-toolbar">
-        <FormAction {...actionProps} />
+        <FormActionComponent {...actionProps} />
+        <ViewModeComponent
+          id="view-mode-toggle-in-edit-nb"
+          area={'edit'}
+        />
       </div>
     );
   }
 
   renderPreview(itemLinks, itemId) {
-    let preview = null;
     const previewClasses = classnames([
       'flexbox-area-grow',
       'fill-height',
       'preview',
-      'campaign-admin__campaign-preview',
```

Comment:
Any reason this was removed?

Reply:
This class was used to hide the preview on smaller screens. Media queries are no longer used for this behaviour, but the ViewModeToggle's state. Therefore the class is redundant.

Reply:
Sure, but removing it is kind of a breaking change - you've already removed the styles for it, but someone else might be relying on it too

Reply:
That makes sense, and is something I haven't had considered. I will revert my change.